tools/ppchcalls.py

```diff
@@ -196,6 +196,15 @@
     63492: 'H_ENTER_NESTED',
     63496: 'H_TLB_INVALIDATE',
     63500: 'H_COPY_TOFROM_GUEST',
+    1120: 'H_GUEST_GET_CAPABILITIES',
+    1124: 'H_GUEST_SET_CAPABILITIES',
+    1136: 'H_GUEST_CREATE',
+    1140: 'H_GUEST_CREATE_VCPU',
+    1144: 'H_GUEST_GET_STATE',
+    1148: 'H_GUEST_SET_STATE',
+    1152: 'H_GUEST_RUN_VCPU',
+    1156: 'H_GUEST_COPY_MEMORY',
+    1160: 'H_GUEST_DELETE',
```

Comment:
Let us add new entries after `1116: 'H_WATCHDOG',` and keep the list sorted for users to check what is supported easily.

Reply:
I've done the changes, please review